plugins/task/privacyconsent/src/Extension/PrivacyConsent.php

```diff
@@ -0,0 +1,258 @@
+                    'url'      => Uri::root(),
+                    'tokenurl' => Route::link('site', 'index.php?option=com_privacy&view=remind&remind_token=' . $token, false, $linkMode, true),
+                    'formurl'  => Route::link('site', 'index.php?option=com_privacy&view=remind', false, $linkMode, true),
+                    'token'    => $token,
+                ];
+
+                $mailer = new MailTemplate('plg_task_privacyconsent.request.reminder', $app->getLanguage()->getTag());
+                $mailer->addTemplateData($templateData);
+                $mailer->addRecipient($user->email);
+
+                $mailResult = $mailer->send();
+
+                if ($mailResult === false) {
+                    return Status::KNOCKOUT;
+                }
+
+                $userId = (int) $user->id;
+
+                // Update the privacy_consents item to not send the reminder again
+                $query->clear()
+                    ->update($db->quoteName('#__privacy_consents'))
+                    ->set($db->quoteName('remind') . ' = 1')
+                    ->set($db->quoteName('token') . ' = :token')
+                    ->where($db->quoteName('id') . ' = :userid')
+                    ->bind(':token', $hashedToken)
+                    ->bind(':userid', $userId, ParameterType::INTEGER);
+                $db->setQuery($query);
+
+                try {
+                    $db->execute();
+                } catch (\RuntimeException $e) {
+                    return Status::KNOCKOUT;
+                }
+            } catch (MailDisabledException | phpmailerException $exception) {
+                return Status::KNOCKOUT;
+            }
+        }
+        $this->logTask('Remind end');
+
+        return Status::OK;
+    }
+
+    /**
+     * Method to delete the expired privacy consents.
+     *
+     * @param   integer    $expire
+     *
+     * @return integer  The routine exit code.
+     *
+     * @since  __DEPLOY_VERSION__
+     * @throws \Exception
+     */
+    private function invalidateExpiredConsents($expire): int
+    {
+        $now    = Factory::getDate()->toSql();
+        $period = '-' . $expire;
+        $db     = $this->getDatabase();
+        $query  = $db->getQuery(true);
+
+        $query->select($db->quoteName(['id', 'user_id']))
+            ->from($db->quoteName('#__privacy_consents'))
+            ->where($query->dateAdd($db->quote($now), $period, 'DAY') . ' > ' . $db->quoteName('created'))
+            ->where($db->quoteName('subject') . ' = ' . $db->quote('PLG_TASK_PRIVACYCONSENT_SUBJECT'))
+            ->where($db->quoteName('state') . ' = 1');
+
+        $db->setQuery($query);
+
+        try {
+            $users = $db->loadObjectList();
+        } catch (\RuntimeException $e) {
+            return Status::KNOCKOUT;
+        }
+
+         // Do not process further if no expired consents found
+         if (empty($users)) {
+            return Status::OK;
+        }
```

Comment:
```suggestion
        // Do not process further if no expired consents found
        if (empty($users)) {
            return Status::OK;
        }
```
Fix PHPCS (last one, I think).